pipelines/concourse.yml

```diff
@@ -1104,6 +1104,27 @@ jobs:
     - get: bbr-sdk-release
     - get: prod
     - get: ci
+  - put: bosh-topgun-worker-deployment
```

Comment:
I think this step should have `tags: [topgun]` based on https://concourse-ci.org/tags-step-modifier.html#tags